tests/workarounds/wrl/wrl/async.h

```diff
@@ -641,7 +641,7 @@ class AsyncBase< TComplete, Details::Nil, resultType, TAsyncBaseOptions > : publ
 
 #ifndef _WRL_DISABLE_CAUSALITY_
             if (SUCCEEDED(hr) &&
-                ::InitOnceExecuteOnce(&gCausalityInitOnce, InitCausality, NULL, NULL))
+                ::InitOnceExecuteOnce(&gCausalityInitOnce, InitCausality, nullptr, nullptr))

```

Comment:
The files under the `workarounds` directory are copied from the SDK; best not to change them unnecessarily